trunk/SUAVE/Methods/Geometry/Two_Dimensional/Cross_Section/Airfoil/compute_airfoil_polars.py

```diff
@@ -0,0 +1,198 @@
+## @ingroup Methods-Geometry-Two_Dimensional-Cross_Section-Airfoil
```

Comment:
We can't have duplicate code. This is pretty much the same as these:
https://github.com/suavecode/SUAVE/tree/feature-vtol/trunk/SUAVE/Methods/Aerodynamics/AERODAS

This is very important to make consistent, if one has a bug no one will think to fix the other and the bug persists

Reply:
Yes, I am aware of this Method in the Aerodynamics folder. The reason I made a new script were

1. the Aerodas method is actually for airfoils - which you extend as a full wing method ( I am not certain about the validity of this.
2. you separated the post-stall and pre stall characteristics making it more cumbersome to interpolate quickly when I want to determine the performance at each airfoil section. I also do not store the results as part of the state but as a property of the airfoil itself

Reply:
Just an FYI,
1. Yes, you can go from sectional properties to 3-D properties. That's not a problem. But I used the low fidelity calcs. The original AERODAS paper used the sectional properties and used them for wind turbines and rotors. I then used the classical aero assumptions to go to 3D.

2. That was because of how the aerodynamics methods/analyses made most sense at the time.